bin/cqlsh.py

```diff
@@ -238,21 +238,6 @@ def find_zip(libprefix):
     except OSError:
         print('\nWarning: Cannot create directory at `%s`. Command history will not be saved.\n' % HISTORY_DIR)
 
-OLD_CONFIG_FILE = os.path.expanduser(os.path.join('~', '.cqlshrc'))
```

Comment:
I dont get it why we are removing this?

Reply:
This was migration code introduced in Cassandra 1.2 when the file location was moved from .cqlshrc -> cqlshrc and .cqlsh_history -> cqlsh_history.  Its time to sunset this compatibility with C* 1.1 file locations.

Reply:
I understand why you plopped them together as a general "fix issues" PR, but  IMO this is essentially two different PR's... one removing this outdated config, and the other fixing issues flagged by pylint.

It's certainly easier to review as separate PR's, and also if for some reason part of this PR has to be reverted later, it's easier if they're separate because only the bit that's broken (config or lint) gets reverted and the other bit stays in.

Just something to think about for the future from a general "code management" perspective as a maintainer.

Reply:
I agree with @jeffwidman , this should be just about pylint. Any removal of obsoletes should be separate. I know it sucks but really, tickets should be just focused on one thing really. I have a feeling like we are returning to this pylint stuff, lets just focus on it and lets get it right so we do not need to ever return to pylint anymore and we can focus on code removal which is obsolete after that.

Reply:
Sure, I can move the OLD_CONFIG and  OLD_HISTORY to a separate PR.

Reply:
@bschoening is there a ticket tracking this? Because that'd be the first step is filing a JIRA. Would certainly be nice to land this in time for 4.1 if still possible